### ISSUE TITLE
[Feature Request] Updater always opens standard browser

### ISSUE THREAD

**Issue by TheBasti82** [master-technology/AssetManagerStudio#22]
I know this isn't a real bug! But, it's annoying! Surely there are ways to download a simple file via HTTPS? It's easy in Python and PHP. If you simply published the program code, others could also add this feature, but that's just ridiculous

**Comment by NathanaelA** [comment]
Yep, it actually isn't that hard to download the updates -- but at this point, I haven't spent the time to automate the download and then running of its installer.   It is on my feature list, but more important features always take up my time.   ;-)
